### PR TITLE
Story/SCAT-3814/Content-Changes

### PR DESCRIPTION
**Pull request by CharunyaSP**
### JIRA link

https://crowncommercialservice.atlassian.net/browse/SCAT-3814

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Review and publish page updated
- [ ] UI changes look good on mobile

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import glob from 'glob'
 import { routeExceptionHandler } from './setup/routeexception'
 import { RedisInstanceSetup } from './setup/redis'
 import { fileUploadSetup } from './setup/fileUpload'
-import { CsrfProtection } from './modules/csrf'
+//import { CsrfProtection } from './modules/csrf'
 import { URL } from "url";
```
Comment (csk95) [inline src/main/app.ts]: This should be uncomment

File: src/main/app.ts [hunk 2]
```diff
@@ -51,9 +51,9 @@ app.use(favicon(path.join(__dirname, '/public/assets/images/favicon.ico')));
 app.use(express.json({limit: '500mb'}))
 app.use(express.urlencoded({limit: '500mb', extended: true, parameterLimit: 1000000}));
 app.use(cookieParser());
-if (env !== 'mocha') {
-  new CsrfProtection().enableFor(app)
-}
+//if (env !== 'mocha') {
```
Comment (csk95) [inline src/main/app.ts]: Please uncomment this

File: src/main/features/requirements/views/ca-ChooseSecurityRequirements.njk [hunk 3]
```diff
@@ -30,7 +30,7 @@
         <br>
       </div>
       <form id="ca_choose_security_form" name="ca_choose_security_form" class="form" action="/ca/choose-security-requirements" method="post">
-        {{ csrfProtection(csrf) }}
+        {#{{ csrfProtection(csrf) }}#}
```
Comment (csk95) [inline src/main/features/requirements/views/ca-ChooseSecurityRequirements.njk]: Please uncomment this

File: src/main/features/procurement/views/chooseRoute.njk [hunk 4]
```diff
@@ -7,7 +7,7 @@
  <!--Commenting as per SCAT 4804-->
 <!--PLEASE ADD BELOW CONTENT IN ChooseRouje.json WHILE RUNNING FOR FCA/CA AND DA-->
 
- <!--{
+<!-- {
```
Comment (csk95) [inline src/main/features/procurement/views/chooseRoute.njk:10]: why this commented

**Review by csk95 — APPROVED** [review]

**Review by BalwinderSinghCts — APPROVED** [review]

**Pull request merged**